tabular/src/autogluon/tabular/models/automm/ft_transformer.py

```diff
@@ -0,0 +1,59 @@
+"""Wrapper of the MultiModalPredictor."""
+import logging
+from .automm_model import MultiModalPredictorModel
+
+logger = logging.getLogger(__name__)
+
+
+class FTTransformerModel(MultiModalPredictorModel):
```

Comment:
There is no need to wrap this class. We can reuse our AutoMMPredictor wrapper but specify the flags.

Reply:
I don't see much harm with doing it this way unless you are concerned with future backwards compatibility. This is easier to access and understand from the user's POV

Reply:
Sounds good. I'm resolving this.